packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -350,35 +347,23 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
     return toTransactionObject(this.kit, this.contract.methods.selectIssuers(phoneHash))
   }
 
-  /**
-   * Reveals the phone number to the issuer of the attestation on-chain
-   * @param phoneNumber The phone number which requested attestation
-   * @param issuer The address of issuer of the attestation
-   */
-  async reveal(phoneNumber: string, issuer: Address) {
-    const accounts = await this.kit.contracts.getAccounts()
-    const publicKey: string = (await accounts.getDataEncryptionKey(issuer)) as any
-
-    if (!publicKey) {
-      throw new Error('Issuer data encryption key is null')
-    }
-
-    const encryptedPhone: any =
-      '0x' +
-      ECIES.Encrypt(
-        Buffer.from(publicKey.slice(2), 'hex'),
-        Buffer.from(phoneNumber, 'utf8')
-      ).toString('hex')
-
-    return toTransactionObject(
-      this.kit,
-      this.contract.methods.reveal(
-        PhoneNumberUtils.getPhoneHash(phoneNumber),
-        encryptedPhone,
+  async requestAttestationFromIssuer(
```

Comment:
Maybe `reveal` is a better name here? or something else. I find this is confusing given that requesting an attestation is already a step in the verification process

Reply:
I'm always open to naming advice, I opted for `requestFromIssuer` because that's what it represents imo (vs. requesting from the smart contract). `reveal` made more sense when it was on-chain as we encrypted the number to the validator, but now we are explicitly requesting from a validator's service

Reply:
+1 to two "request"s being confusing. In some sense, we are revealing our phone number here to the issuer. Alternatively, maybe the original request is more of a commitment to request attestations?

Reply:
I like commit